test/e2e/benchmark/throughput.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/v2/pkg/appconsts"
+	"github.com/celestiaorg/celestia-app/v2/test/e2e/testnets"
+	"github.com/celestiaorg/celestia-app/v2/test/util/testnode"
+)
+
+const seed = 42
+
+func main() {
+	if err := E2EThroughput(); err != nil {
+		log.Fatalf("--- ERROR Throughput test: %v", err.Error())
+	}
+}
+
+func E2EThroughput() error {
+	os.Setenv("KNUU_NAMESPACE", "test")
+
+	latestVersion, err := testnets.GetLatestVersion()
+	testnets.NoError("failed to get latest version", err)
+
+	log.Println("=== RUN E2EThroughput", "version:", latestVersion)
+
+	// create a new testnet
+	testnet, err := testnets.New("E2EThroughput", seed, testnets.GetGrafanaInfoFromEnvVar())
+	testnets.NoError("failed to create testnet", err)
+
+	log.Println("Cleaning up testnet")
```

Comment:
I expect this log will be printed immediately but the testnet will be cleaned up at the end of the test run. Proposal to move this log line inside the testnet.Cleanup() implementation and delete it here. 

```suggestion
```

Reply:
i think cleaning up testnet describes it correctly, since it implies that the process of cleaning up is ongoing. i'm happy to move it inside the cleanup though if you think it helps with readability/understanding.

Reply:
In the original e2e throughput test (of the main branch), the method for informing the clean up of the testnet is different. Could you please elaborate on the reasons for this change from the original test? Here’s the specific section of the code for reference: https://github.com/celestiaorg/celestia-app/blob/a8e3085c98301d0dc0c2a5c86880b5fc51f904de/test/e2e/throughput_test.go#L39-L42

Reply:
What you could do is to wrap the testnet.Cleanup() call inside an anonymous function, and make it like
```go
defer func() { 
		t.Log("Cleaning up testnet") 
		testnet.Cleanup()}()
```